src/conf.py

```diff
@@ -73,9 +73,10 @@
     # TODO Update this when tutorials have a proper home.
```

Comment:
Nothing to do there. Delete line.

Reply:
Eh, there is something to do, but will remove it for now I guess.  63f6708